extensions/adobe/experience/adcloud/searchadvertisingcampaign.schema.json

```diff
@@ -0,0 +1,166 @@
+{
+  "meta:license": [
+    "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/adcloud/searchadvertisingcampaign",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Search Advertising Campaign",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:auditable": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "description": "Search Advertising Campaigns is a set of ad groups (ads, keywords, and bids) that share a budget, location targeting, and other settings.",
+  "definitions": {
+    "searchadvertisingcampaign": {
+      "properties": {
+        "xdm:campaignId": {
+          "title": "Campaign ID",
```

Comment:
Given this field is defined as a core "xdm" field, does it need to have the exact same title as other locations? I see in the adcloud/campaign schema this field has a title of "Campaign Identifier".

Reply:
Modified all ID fields title to have identifier fully spelled rather than ID.